src/misc.c

```diff
@@ -334,6 +370,8 @@ static const JSCFunctionListEntry tjs_misc_funcs[] = {
     JS_CFUNC_DEF("exepath", 0, tjs_exepath),
     JS_CFUNC_MAGIC_DEF("print", 1, tjs_print, 0),
     JS_CFUNC_MAGIC_DEF("printError", 1, tjs_print, 1),
+    JS_CFUNC_MAGIC_DEF("alert", 1, tjs_print, 1),
+    JS_CFUNC_DEF("prompt", 0, tjs_prompt),
```

Comment:
this needs to be 2, since it takes 2 arguments

Reply:
Both arguments are optional.